upstream/CVE-2024-3744.json

```diff
@@ -0,0 +1,50 @@
+{
+  "id": "CVE-2024-3744",
+  "modified": "2024-05-08T16:02:57Z",
+  "published": "2024-05-08T16:02:57Z",
+  "summary": "azure-file-csi-driver discloses service account tokens in logs",
+  "details": "A security issue was discovered in azure-file-csi-driver where an actor with access to the driver logs could observe service account tokens. These tokens could then potentially be exchanged with external cloud providers to access secrets stored in cloud vault solutions. Tokens are only logged when TokenRequests is configured in the CSIDriver object and the driver is set to run at log level 2 or greater via the -v flag.\n\n",
+  "affected": [
+    {
+      "package": {
+        "ecosystem": "kubernetes",
+        "name": "/azure-file-csi-driver"
+      },
+      "severity": [
+        {
+          "type": "CVSS_V3",
+          "score": "CVSS:3.1/AV:L/AC:L/PR:L/UI:N/S:C/C:H/I:N/A:N"
+        }
+      ],
+      "ranges": [
+        {
+          "type": "SEMVER",
+          "events": [
+            {
+              "introduced": "1.29.0"
+            },
+            {
+              "last_affected": "1.29.3"
+            },
+            {
+              "introduced": "1.30.0"
```

Comment:
ditto
>Affected Versions
> - azure-file-csi-driver <= v1.29.3
> - azure-file-csi-driver v1.30.0

```suggestion
              "introduced": "0"
```

Reply:
@knqyf263  why you think `introduced: 1.30.0` should be change to `0` as all versions < `1.30` already covered by prev. entry:


`introduced: 0`


`fixed: 1.29.4`

Reply:
Oh, yes. I'm totally wrong.